Huthman King: Project Week - Front End

kinghuthman opened the pull request:
Trello Set Up:

- [x]  Create a Trello account
- [x]  Create a new board called "Lambda Notes - {Your Name}"
- [x]  Create lists titled backlog,To Do, In Progress, and Done
- [x]  Fill in the To Do list with the MVP features listed below
- [x]  Fill in the backlog list with all the extra features listed below
- [x]  Share your board with the project manager that has been assigned to you. If you have not been assigned yet, reach out to your lead PM for guidance
- [x]  Add your Trello URL to your project's README.md file. Commit the change, push it to your repository & submit a pull request

MVP Features:
 

- [x] Display a list of notes
- [x]  Create a note with a title and content
- [x]  View an existing note
- [x]  Edit an existing note
- [x]  Delete an existing note



@bradiowave

Commit:
Huthman King: Project Week - Front End
initial commit

Commit:
Initial commit  - 2
added trello link to readme
added react router dom

Commit:
Display a list of notes
'Display a list of Notes
Created Notecard component
Created Noteview
Created Note List (displays dummy data created)
wrapped app with router within index
Imported components to App js file

Commit:
Sidebar & Styling

Commit:
Morning Work
added ellipses through npm
Slight styling touch up
Created notetext component
imported notetext to notecard
added reactrouter
Created addnote component
Created editnote component
imported withrouter to noteview
updated app.js :)

Commit:
End of Day
Implemented Add functionality
Implemented Edit functionality
Implemented Deleted functionality and added delete modal
Styled

Commit:
Clean Up

Bradiowave commented:
What looks great:
- MVP is complete!!!
- Followed the design specs
- Nice separation of components
- App is instantly responsive
- Code is well formatted and easily readable
- Nice wrapping of title and use of ellipses when textBody is long

What could be improved:
- Could use more commits. It's good to commit every time you make a change that you don't want to lose.
- Use separate .css files for each component. The single App.css is too cluttered

Commit:
Back-End connected
Connected front-end to back end
bugs when adding a note, forced to refresh to see new note
unable to see individual notes content
unable to edit individual note

Commit:
Noteview Works
still unable to edit/delete
buggy post

Commit:
Still buggy

Commit:
Cleaned up bugs
need to refresh to see changes, but functional

Commit:
Clean up

Commit:
Animation

kinghuthman committed:
API implemnted

kinghuthman committed:
Updates
Updated edit axios call
changed api to local host back end url

kinghuthman committed:
key added to note list to remove error

kinghuthman committed:
Updated readme

kinghuthman committed:
updated readme
